### PR TITLE
Fixed wrong database url in console log and profile data  not saving bug.

### LINKED ISSUE #67: Complete Profile Section
After signing up through google it redirect me to complete my profile section.But Its taking a lot of time to save my details.In the UI part its still showing saving profile.

Screenshot from 2021-05-18 02-43-00

### PR DESCRIPTION
**Pull request by rijusougata13**
I have added two new field in .env.sample .  In the functions/auth.js we are fetching wrong database url from .env file , I have replaced it  with correct one. 

## Error before fix
Screenshot from 2021-06-06 13-13-54

## Code 
```
const firebaseConfig = {
  apiKey: process.env.REACT_APP_FIREBASE_API_KEY,
  authDomain: process.env.REACT_APP_AUTH_DOMAIN,
  databaseURL: process.env.REACT_APP_DATABASE_URL,
  projectId: process.env.REACT_APP_FIREBASE_PROJECT_ID,
  storageBucket: `${process.env.REACT_APP_FIREBASE_PROJECT_ID}.appspot.com`,
  messagingSenderId: process.env.REACT_APP_FIREBASE_MESSAGING_SENDER_ID,
  appId: process.env.REACT_APP_FIREBASE_APP_ID,
  measurementId: process.env.REACT_APP_FIREBASE_MEASUREMENTID,
};

```
## Related Issue
This pr will also fix #67 

## How Has This Been Tested?
After fixing this there is no console error ("wrong database url").

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by Niweera — APPROVED** [review]

File: .firebaserc
Comment (Niweera) [inline .firebaserc:3]: I have a doubt here. Is `.firebaserc` file read by React? I don't think so. It is created by 

```bash
$ firebase init
```
command and it is read only by `firebase` cli. So changing this to an env variable is ill-advised I think. But I need a second opinion on this. @thisuraseniya, @shehand?

Comment (rijusougata13) [reply .firebaserc:3]: let me see if get any documentation or not. maybe others can help in this case.

File: .gitignore
Comment (shehand) [inline .gitignore:36]: What this means? You committed that file and gitignore the file same time? Any specific reason?

File: .firebaserc
Comment (shehand) [reply .firebaserc:3]: I'm also having the same doubt. I think this will affect to GitHub actions(if we have them here to deploy) also since firebaserc is used in GitHub actions in firebase deployment. Better to revert this change back to original.

**Review by shehand — CHANGES_REQUESTED** [review]

File: .gitignore
Comment (rijusougata13) [reply .gitignore:36]: I thought that it will differ for every member that project id , so i thought it would be better to use env varible or put it in gitignore . I removed firebaserc from gitignore , now there will be no problem

File: .firebaserc
Comment (rijusougata13) [reply .firebaserc:3]: ok changing it back.

**Review by thisuraseniya — APPROVED** [review]

**Comment by thisuraseniya** [comment]
Will merge after @shehand puts his review :D

**Comment by rijusougata13** [comment]
> Will merge after @shehand puts his review :D

sure!!

File: .firebaserc
Comment (shehand) [inline .firebaserc:3]: Ok, Now i'm seeing the issue here. let's ignore the firebaserc file. Revert this change back.

**Review by shehand — APPROVED** [review]

File: .firebaserc
Comment (rijusougata13) [reply .firebaserc:3]: can I do this like, change firebaserc to the default 'codelabz-dev' and push the commit ,  then I can add this file to gitignore and push. I think it will solve the problem , our main codebase has default value as "codelabz-dev" , and if anyone change this file it will not effect our main codebase.

Comment (rijusougata13) [reply .firebaserc]: @shehand  changed as mentioned above.

**Review by shehand — APPROVED** [review]

**Pull request merged**
